docs/UserGuide.adoc

```diff
@@ -190,165 +186,154 @@ Returns any person having birthday of `17-07`
 
 __Constraints:__
 
+Note the following constraint when trying to find contacts:
+
 * Only full words will be matched e.g `Han` will not match `Hans`, `friend` will not match `friends` and `1707` will not match `17-07`
 
+
 ==== Deleting contacts : `delete`
 
-Deleting contacts requires just one simple command:
+If you need to clean up your contact list, just delete all those redundant contacts!
 
 Format : `delete INDEX`
 
-
-All information associated with the contact will be wiped away.
-
-To illustrate:
+Examples:
 
 * `list` +
 `delete 2` +
-Deletes the 2nd person in the address book.
+Deletes the 2nd contact in the list.
 
 * `find Betsy` +
 `delete 1` +
-Deletes the 1st person in the results of the `find` command.
+Deletes the 1st contact in the result list of the `find` command.
 
 __Constraints__
 
-****
-* Deletes the person at the specified `INDEX`.
-* The index refers to the index number shown in the most recent listing.
+Note the following constraints when trying to delete contacts:
+
+* Deletes the contact at the specified index.
+* INDEX refers to the index number shown in the most recent contact listing.
 * The index *must be a positive integer* 1, 2, 3, ...
-****
 
-[coming in v2.0]
+
 ==== Tagging multiple contacts : `tag` +
 
 Tags are a useful way of grouping and labeling contacts. But editing them manually, one by one, can be a tedious process.
+Instead, you can use the tag command to tag multiple contacts simultaneously.
 
-. Instead, use:
-Format: `tag [INDEX]… t/[TAG]…`
-
-. Multiple contacts will be tagged by the tag(s) set in the parameter.
-
-. The Contact Cards will be updated to display the new tags.
+Format: `tag INDEX… t/[TAG]…`
 
 [TIP]
 You can add multiple tags to multiple contacts by repeating the tag prefix.
 
-To illustrate, the command:
+Example:
 
 `tag 1, 2, 3 t/friend t/family` +
-Results in all contacts in indices 1, 2 and 3 to be tagged with both [friend] and [family].
+Results in all contacts in indices 1, 2 and 3 being tagged with both [`friend`] and [`family`].
 
 __Constraints__
 
-****
-* Each index MUST be valid according to the current list of contacts.
+Note the following constraints when trying to add tags:
```

Comment:
a plus sign in front does not break a line,
do you want the format to be a `+` in front of every line in constraint?

____TEST____
+Note this
+*format

versus

Note +
this +

Reply:
Nope, no plus signs in the actual formatting except wherever it's needed to break lines. That's probably a stray but it doesn't seem to show up in the actual document. You can delete it or leave it.

Reply:
i see, then i will put + sign at end of line to break the sentence. but this is under the tag command, i am referencing it, so you may want to change it. thanks!

Reply:
Sorry my bad, i mix up the GitHub changes with the code.

Reply:
no worries, easy to miss these things